extensions/wikia/Search/classes/Query/Select.php

```diff
@@ -3,103 +3,120 @@
  * Class definition for Wikia\Search\Query\Select
  */
 namespace Wikia\Search\Query;
-use Wikia\Search\MediaWikiService, Wikia\Search\Utilities, Sanitizer, Solarium_Query_Helper;
+
+use Wikia\Search\MediaWikiService, Sanitizer, Solarium_Query_Helper;
+
 /**
  * This class is responsible for abstracting the different ways we express queries.
+ *
  * @author relwell
  */
-class Select
-{
+class Select {
 	/**
 	 * The pristine value passed during construct with no alterations
+	 *
 	 * @var string
 	 */
 	protected $rawQuery;
-	
+
 	/**
 	 * Any possible namespace prefix -- has to be an actual prefix, though.
+	 *
 	 * @var string
 	 */
 	protected $namespacePrefix;
-	
+
 	/**
 	 * Keeps us from checking for namespace more than once.
+	 *
 	 * @var boolean
 	 */
 	protected $namespaceChecked = false;
-	
+
 	/**
 	 * If we have a namespace query, then an int. Otherwise null.
+	 *
 	 * @var int
 	 */
 	protected $namespaceId;
-	
+
 	/**
 	 * This is the query string *without* any namespace prefix.
 	 * We use this for passing to Solr queries.
+	 *
 	 * @var string
 	 */
 	protected $strippedQuery;
-	
+
 	/**
 	 * This is our original query with any XSS vulns and tags removed.
+	 *
 	 * @var string
 	 */
 	protected $sanitizedQuery;
-	
+
 	/**
 	 * MediaWikiService, encapsulates MediaWiki logic
+	 *
 	 * @var \Wikia\Search\MediaWikiService
 	 */
 	protected $service;
-	
+
 	/**
 	 * Constructor method. Requires the original query string.
 	 * Sets this value as protected attribute rawQuery.
 	 * This is raw user input, so you should use a public accessor to get the right kind of query.
+	 *
 	 * @param string $queryString
 	 */
 	public function __construct( $queryString ) {
 		$this->rawQuery = $queryString;
 	}
-	
+
 	/**
 	 * Lazy-loads a sanitized version of the user input.
+	 *
 	 * @return string
 	 */
 	public function getSanitizedQuery() {
 		if ( $this->sanitizedQuery == null ) {
-			$this->sanitizedQuery = html_entity_decode( Sanitizer::stripAllTags( $this->rawQuery ), \ENT_COMPAT, 'UTF-8');
+			$this->sanitizedQuery =
+				html_entity_decode( Sanitizer::stripAllTags( $this->rawQuery ), \ENT_COMPAT, 'UTF-8' );
 		}
+
 		return $this->sanitizedQuery;
 	}
-	
+
 	/**
 	 * Returns the value we use when querying Solr, with the appropriate special characters escaped.
 	 * Word limit allows us to optionally set a maximum number of words in a query
-	 * @TODO applying word limit here is clumsy at best and suggests it's about time we implement some class abstraction
-	 * and hierarchy around what kinds of queries we're using where. this would be good for an actual edismax query, not so 
-	 * great for video suggest or direct solr queries, for instance.
+	 *
+	 * @TODO applying word limit here is clumsy at best and suggests it's about time we implement some class
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)